cmd/cli/config/config.go

```diff
@@ -103,6 +103,15 @@ func getConfigPath(p ...string) (string, error) {
 		}
 	}
 
-	cfgPath = path.Join(cfgPath, configName)
+	info, err := os.Stat(cfgPath)
```

Comment:
we have the `util.DirectoryExists()` utility, so this checks can be replaced with one line